pennylane/ops/functions/iterative_qpe.py

```diff
@@ -0,0 +1,71 @@
+# Copyright 2018-2023 Xanadu Quantum Technologies Inc.
+
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+
+#     http://www.apache.org/licenses/LICENSE-2.0
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""
+This module contains the qml.iterative_qpe function.
+"""
+
+import numpy as np
+import pennylane as qml
+
+
+def iterative_qpe(base, estimation_wire, iters):
```

Comment:
Maybe a question for @trbromley, should we add this to any of the major documentation pages? For example the functions subsection in [here](https://docs.pennylane.ai/en/stable/introduction/operations.html)

Reply:
Yes it's a good point, I do feel like it's missing from somewhere. Yep we could put it [here](https://docs.pennylane.ai/en/stable/introduction/operations.html#operator-to-operator-functions).